ports/esp32/modesp32.c

```diff
@@ -153,6 +182,7 @@ STATIC const mp_rom_map_elem_t esp32_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_wake_on_ext1), MP_ROM_PTR(&esp32_wake_on_ext1_obj) },
     { MP_ROM_QSTR(MP_QSTR_raw_temperature), MP_ROM_PTR(&esp32_raw_temperature_obj) },
     { MP_ROM_QSTR(MP_QSTR_hall_sensor), MP_ROM_PTR(&esp32_hall_sensor_obj) },
+    { MP_ROM_QSTR(MP_QSTR_espidf_heap_info), MP_ROM_PTR(&esp32_espidf_heap_info_obj) },
```

Comment:
Since the function is in the `esp32` module, maybe just `heap_info` is enough, or `idf_heap_info`?

Reply:
`idf_heap_info` would be fine with me, I just wanted to make sure someone new doesn't think this is the MP heap...

Reply:
sounds good